HabeebAwawda/task.py

```diff
@@ -0,0 +1,42 @@
+class Task:
+    # one
+
+    task_id = 0
+    title = ""
+    description = ""
+    due_date = ""
+    status = ""
+
+    def __init__(self, title, description, due_date, status):
+        Task.task_id += 1
+        self.task_id = Task.task_id
+        self.title = title
+        self.description = description
+        self.due_date = due_date
+        self.status = status
+
+    def update_details(self, title, description, due_date, status):
```

Comment:
This method is good for updating task attributes but consider using self for method chaining.